service/src/main/java/uk/gov/hmcts/reform/fpl/model/docmosis/welshtranslation/DocmosisTranslationRequest.java

```diff
@@ -19,7 +19,7 @@
     private String description;
     private DocmosisTranslateLanguages translate;
     private DocmosisWelshLayout layout;
-    private int wordCount;
+    private long wordCount;
```

Comment:
Does this need to be long?

Reply:
the stream.count() in the wordCalculator returns normally a long, so rather than downcasting to an int (casting is always ugly) I've change it to long.